scripts/migrations/src/migrations/20241216T160549_event_registry.ts

```diff
@@ -0,0 +1,27 @@
+import { Kysely } from "kysely";
+
+/**
+ * The up function is called when you update your database schema to the next version and down when you go back to previous version.
+ * The only argument for the functions is an instance of Kysely<any>. It's important to use Kysely<any> and not Kysely<YourDatabase>.
+ * ref: https://kysely.dev/docs/migrations#migration-files
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function up(db: Kysely<any>): Promise<void> {
+    const CHAIN_ID_TYPE = "integer";
+
+    await db.schema
+        .createTable("events")
```

Comment:
Does using `last_events` as the table name fit better to this use case? It feels weird having a table named `events` with a `chainId` primary key.

```suggestion
        .createTable("last_events")
```